### PR TITLE
[2] Random corrections for device manager PR

### PR DESCRIPTION
**Pull request by patrickelectric**
Based over https://github.com/bluerobotics/ping-viewer/pull/499

### CODE REVIEW

**Comment by jaxxzer** [comment]
@patrickelectric why do you recommend svg over png?

File: qml/FirmwareUpdate.qml [hunk 1]
```diff
@@ -23,7 +23,8 @@ Item {
             running = true
             flashProgress.value = progress
         }
-        onMessageChanged: {
+        onStateChanged: {
+            print("Flash state:", state)
```
Comment (jaxxzer) [inline qml/FirmwareUpdate.qml]: stale debug?

Can you give a longer commit message, it's vague right now.

**Comment by jaxxzer** [comment]
fix/merge #499 first

File: qml/FirmwareUpdate.qml (hunk 1 above)
Comment (patrickelectric) [reply qml/FirmwareUpdate.qml]: Removed.

**Pull request merged**